### DISCUSSION TITLE
Is there a Bookmarklet to Just Turn it Off?

### DISCUSSION THREAD

**Discussion by Overreationary** [3kh0/ext-remover#560] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
As you can see in the video, I can use point-blank to run the on-and-off bookmarklet, which basically turns on and off GoGuardian constantly. This works for unblocking things, but it's so laggy with an extension being turned on and off, that it's not viable. But if someone had a bookmarklet code to do the same thing, and past the extension code in there just like I do in the video, but instead of constantly turning it on and off, they just turn it off, then I think we would find something to disable goguardian. 
[Screen recording 2023-03-08 7.18.19 PM.webm](https://user-images.githubusercontent.com/93745423/223891106-e25d3c2e-efb4-4738-ac8b-503a51a75ccc.webm)

**Comment by obkduishinijf** [comment]
unless you're on version 101 or below, or goguardian was updated by your admin, then you can't. google fixed it, and goguardian removed the one XML file we could use to disable it.

**Reply by Aka-but-nice** [reply]
Man it is not a xml file bro, it is a html file that used url params in a way that used innerHTML to control the html on the page.
